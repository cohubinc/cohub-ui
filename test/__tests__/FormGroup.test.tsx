import React from "react";
import { mount } from "enzyme";

import { FormGroup } from "dist";

it("renders without crashing", async () => {
  const wrapper = mount(
    <FormGroup direction="horizontal">
      <div>test</div>
    </FormGroup>
  );

  expect(wrapper).toMatchSnapshot();
});
