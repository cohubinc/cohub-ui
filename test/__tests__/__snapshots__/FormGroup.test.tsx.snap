// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<FormGroup
  direction="horizontal"
>
  <div
    className="FormGroup-module_horizontal__M22Uj undefined"
  >
    <div>
      test
    </div>
  </div>
</FormGroup>
`;
